lighthouse-core/test/audits/image-aspect-ratio-test.js

```diff
@@ -11,9 +11,14 @@ const assert = require('assert').strict;
 /* eslint-env jest */
 
 function generateImage(clientSize, naturalSize, props, src = 'https://google.com/logo.png') {
-  const image = {src, mimeType: 'image/png'};
-  Object.assign(image, clientSize, naturalSize, props);
-  return image;
+  return {
+    src,
+    mimeType: '...',
```

Comment:
```suggestion
    mimeType: 'image/png',
```

a reason for the change?

Reply:
oh I see it was [suggested here](https://github.com/GoogleChrome/lighthouse/pull/11707#discussion_r535488917), I'm guessing he meant to refer to whatever was there already rather than replace with a literal. let's leave it as `image/png` 👍 (though kinda hilarious that it doesn't matter as long as it's not an svg